packages/components/src/date-time/stories/date.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import DatePicker from '../date';
+
+const meta: ComponentMeta< typeof DatePicker > = {
+	title: 'Components/DatePicker',
+	component: DatePicker,
+	argTypes: {
+		currentDate: { control: 'date' },
+		events: { control: { type: null } },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof DatePicker > = ( args ) => (
+	<DatePicker { ...args } />
+);
+
+export const Default: ComponentStory< typeof DatePicker > = Template.bind( {} );
+
+function daysFromNow( days: number ) {
+	const date = new Date();
+	date.setDate( date.getDate() + days );
+	return date;
+}
```

Comment:
Is it worth reducing some of the code duplication between the Date and DateTimePicker stories?

Reply:
I'll move `daysFromNow` and `isWeekend` to a `utils` file. I don't think the other repeated bits (e.g. `WithEvents`, `WithInvalidDates`) makes senes to de-duplicate because they have different types and introducing type parameters here might make it too confusing for future developers to change the examples.